src/data_management/wms_layers/WmsLayerForm.js

```diff
@@ -370,6 +370,20 @@ class WmsLayerFormModel extends Component {
           }
           validators={[this.maxZoomValidator]}
         />
+        <SpatialBoundsField
+          name="wmsLayerSpatialBounds"
+          title={<FormattedMessage id="wms_layer_form.spatial_bounds" />}
+          subtitle={<FormattedMessage id="wms_layer_form.add_spatial_bounds" />}
+          initial = {
+            (
+              currentWmsLayer &&
+              currentWmsLayer.spatial_bounds
+            ) || null
+          }
+          validators={[
+            (fieldValue) => spatialBoundsValidator(fieldValue)
```

Comment:
`(param) => someFunc(param)` is exactly the same as just `someFunc`; so this can be `validators={[spatialBoundsValidator]}`

Reply:
Nice